sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -1,5 +1,5 @@
 {%- if CONFIGURED_ARCH == "armhf" and MULTIARCH_QEMU_ENVIRON == "y" %}
-FROM multiarch/qemu-user-static:x86_64-arm-5.2.0-2 as qemu
+FROM multiarch/qemu-user-static:x86_64-arm-6.1.0-8 as qemu
```

Comment:
>6.1.0-8 [](http://example.com/codeflow?start=43&length=7)

This version is same as below aarch64. Will it always be the same in future? If yes, use a constant?

Reply:
I think it makes sense to have it be the same for both armhf and aarch64. It may or may not need changing in the future.

I'll make a variable for this.

Reply:
Docker doesn't like setting an `ARG` before any `FROM` commands, which leaves two alternatives:

* Restructure the armhf and arm64 conditionals to bring in the Debian image first, then set the `ARG`, then bring in the qemu image.
* Make it a Jinja2 variable, and specify the qemu version in `Makefile.work` when parsing the template.

The second might be nicer, since that'll allow setting a single version to be used across all slave images (Stretch, Buster, and Bullseye). If that's fine, I'll change it to that.